packages/algob/src/lib/exceptions.ts

```diff
@@ -0,0 +1,9 @@
+import type { Network } from "../types";
+
+export function checkAlgorandUnauthorized (e: any, n: Network): boolean { // eslint-disable-line @typescript-eslint/no-explicit-any
+  if (e instanceof Error && (e.message === 'Unauthorized')) {
```

Comment:
This can match any exception that is cryptic

Reply:
Yes, I know. But I don't know any better way how to do it. I can add more `e` checks, but there always will be a risk.
On the other hand, I don't know any other dependency which can rise this exception, because we are only connecting to algroand.
Do you have better idea?

Reply:
I've added more "error decoding"